resources/views/hardware/bulk.blade.php

```diff
@@ -21,6 +21,9 @@
 
     <div class="callout callout-warning">
       <i class="fas fa-exclamation-triangle"></i> {{ trans_choice('admin/hardware/form.bulk_update_warn', count($assets), ['asset_count' => count($assets)]) }}
+        @if (count($models) > 0)
+            made up of {{count($models)}} models
+        @endif 
```

Comment:
We should use a translation for this string but also....the translation string above this ends with a `.` so we might need to make a new translation string that is like `bulk_update_with_fields` or something.

![example](https://user-images.githubusercontent.com/1141514/230484084-64dff3a6-8d7f-4b84-b9ec-a4154602a09a.png)